app/geo/gui/src/ts/twxplore/gui/geo/components/map/MapLayout.tsx

```diff
@@ -151,15 +151,17 @@ const MapLayoutImpl: React.FunctionComponent = () => {
           <Grid
             container
             item
+            direction="column"
             spacing={0}
             justify={"flex-start"}
             className={classes.drawerGridContainer}
           >
-            <Grid item xs={12}>
+            <Grid item xs>
               <Typography align={"center"}>Filters</Typography>
             </Grid>
-
-            <FilterPanel />
+            <Grid container item direction="column">
```

Comment:
Not sure why you need another container here. The FilterPanel should have its own Grid container if it needs one.